genny/auth/auth.go

```diff
@@ -0,0 +1,152 @@
+package auth
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/gobuffalo/buffalo-auth/transformer"
+	"github.com/gobuffalo/buffalo/generators"
+	"github.com/gobuffalo/buffalo/meta"
+	"github.com/gobuffalo/genny"
+	"github.com/gobuffalo/genny/movinglater/plushgen"
+	"github.com/gobuffalo/packr"
+	"github.com/gobuffalo/plush"
+	"github.com/markbates/inflect"
```

Comment:
try to start using gobuffalo/flect if you can. if you can't, please open a PR. :)